include/libopencm3/stm32/h7/rcc.h

```diff
@@ -102,7 +94,37 @@ LGPL License Terms @ref lgpl_license
 #define RCC_DCKCFGR2              MMIO32(RCC_BASE + 0x90)
 /**@}*/
 
-/** @addtogroup rcc_cfgr_values RCC_CFGR_VALUES
+/** @defgroup rcc_cr_values RCC_CR Values
+ * @ingroup rcc_registers
+@{*/
+#define RCC_CR_PLL3AIRDY          BIT29
+#define RCC_CR_PLL3AION           BIT29
```

Comment:
Where did these names come from?  RM0433 has them as PLL3RDY and PLL3ON?  (Also, you need to use BIT28 for one of them)